### PR TITLE
[Doc] Add Chinese doc for inference.md

### PR DESCRIPTION
**Pull request by MengzhangLI**
Add Chinese doc for inference.md

### CODE REVIEW

File: docs_zh-CN/inference.md [hunk 1]
```diff
@@ -1 +1,100 @@
-## 用预训练模型进行推理
+## 使用预训练模型推理
+
+我们提供测试脚本来评估完整数据集（Cityscapes, PASCAL VOC, ADE20k 等） 上的结果，同时为了对其他项目的整合更容易，也提供一些高级 API。
+
+### 测试一个数据集
+
+- 单卡 GPU
+- 单节点 多卡 GPU
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:8]:
```suggestion
- 单节点多卡 GPU
```

File: docs_zh-CN/inference.md [hunk 2]
```diff
@@ -1 +1,100 @@
-## 用预训练模型进行推理
+## 使用预训练模型推理
+
+我们提供测试脚本来评估完整数据集（Cityscapes, PASCAL VOC, ADE20k 等） 上的结果，同时为了对其他项目的整合更容易，也提供一些高级 API。
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:3]:
```suggestion
我们提供测试脚本来评估完整数据集（Cityscapes, PASCAL VOC, ADE20k 等） 上的结果，同时为了使其他项目的整合更容易，也提供一些高级 API。
```

File: docs_zh-CN/inference.md [hunk 3]
```diff
@@ -1 +1,100 @@
...
+### 测试一个数据集
+
+- 单卡 GPU
+- 单节点 多卡 GPU
+- 多节点
+
+您可以使用以下命令来测试一个数据集。
+
+```shell
+# 单卡GPU 测试
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:14]:
```suggestion
# 单卡 GPU 测试
```

File: docs_zh-CN/inference.md [hunk 4]
```diff
@@ -1 +1,100 @@
...
+python tools/test.py ${配置文件} ${检查点文件} [--out ${结果文件}] [--eval ${评估指标}] [--show]
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:15]: 可以参考一下 mmdet 中，检查点文件怎么翻译的

File: docs_zh-CN/inference.md [hunk 5]
```diff
@@ -1 +1,100 @@
...
+./tools/dist_test.sh ${配置文件} ${检查点文件} ${GPU数目} [--out ${结果文件}] [--eval ${评估指标}]
+```
+
+可选参数:
+
+- `RESULT_FILE`: pickle 格式的输出结果的文件名，如果不专门指定，结果将不会被专门保存成文件。
+- `EVAL_METRICS`: 在结果里将被评估的指标。这主要取决于数据集，  `mIoU`  对于所有数据集都可获得，像 Cityscapes 数据集可以通过 `cityscapes` 命令来专门评估，就像标准的 `mIoU`一样。
+- `--show`: 如果被指定，分割结果将会在一张图像里画出来并且在另一个窗口展示。它仅仅是用来调试与可视化，并且仅针对单卡 GPU 测试。请确认 GUI 在您的环境里可用，否则您也许会遇到报错 `cannot connect to X server`
+- `--show-dir`: 如果被指定，分割结果将会在一张图像里画出来并且保存在指定文件夹里。它仅仅是用来调试与可视化，并且仅针对单卡GPU测试。使用该参数时，您的环境不需要 GUI。
+- `--eval-options`: 评估时的可选参数，当设置 `efficient_test=True` 时，它将会保存中间结果至本地文件里以节约CPU内存。请确认您本地硬盘有足够的存储空间（大于20GB）。
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:27]:
```suggestion
- `--eval-options`: 评估时的可选参数，当设置 `efficient_test=True` 时，它将会保存中间结果至本地文件里以节约 CPU 内存。请确认您本地硬盘有足够的存储空间（大于20GB）。
```

File: docs_zh-CN/inference.md [hunk 6]
```diff
@@ -1 +1,100 @@
...
+
+4. 使用4卡 GPU 测试 PSPNet，并且在标准 mIoU 和 cityscapes 指标里评估模型。
+
+    ```shell
+    ./tools/dist_test.sh configs/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes.py \
+        checkpoints/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth \
+        4 --out results.pkl --eval mIoU cityscapes
+    ```
+
+   注意：在 cityscapes mIoU 和我们的 mIoU 指标会有一些差异 (~0.1%) 。原因是因为 cityscapes 会以默认的类的大小去平均每一类。而我们不去平均，是更简单的，适用于所有数据集的方法。
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:65]: 而我们不去平均？

File: docs_zh-CN/inference.md [hunk 7]
```diff
@@ -1 +1,100 @@
...
+    ```shell
+    ./tools/dist_test.sh configs/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes.py \
+        checkpoints/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth \
+        4 --format-only --eval-options "imgfile_prefix=./pspnet_test_results"
+    ```
+
+   您会在文件夹 `./pspnet_test_results` 里得到生成的 png 文件。
+   您也许可以运行 `zip -r results.zip pspnet_test_results/` 并提交 zip 文件给 [evaluation server](https://www.cityscapes-dataset.com/submit/)。
+
+6. CPU memory efficient test DeeplabV3+ on Cityscapes (without saving the test results) and evaluate the mIoU.
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:89]:
```suggestion
```

File: docs_zh-CN/inference.md [hunk 8]
```diff
@@ -1 +1,100 @@
...
+6. 在 Cityscapes 数据集上测试 CPU 内存高效率版本的 DeeplabV3+ (没有保存测试结果) 并且评估 mIoU。
```
Comment (Junjun2016) [inline docs_zh-CN/inference.md:90]:
```suggestion
6. 在 Cityscapes 数据集上使用 CPU 高效内存选项来测试 DeeplabV3+ `mIoU` 指标 (没有保存测试结果)。
```

Comment (xvjiarui) [inline docs_zh-CN/inference.md:90]:
```suggestion
6. 在 Cityscapes 数据集上测试 CPU 内存高效率版本的 DeeplabV3+ (不保存测试结果) 并且评估 mIoU。
```

File: docs_zh-CN/inference.md (hunk 4 above)
Comment (MengzhangLI) [reply docs_zh-CN/inference.md:15]: > 可以参考一下 mmdet 中，检查点文件怎么翻译的

I just searched keyword "`checkpoint`" in MMDet, seems it doesn't release Chinese tutorial docs. 

Thus currently I kept the word `检查点` together with its English word `(checkpoints)`.

**Comment by Junjun2016** [comment]
Hi @MengzhangLI 
Please fix the conflicts.

File: docs_zh-CN/inference.md (hunk 6 above)
Comment (Junjun2016) [reply docs_zh-CN/inference.md:65]: ？

**Pull request merged**